src/api/app/controllers/concerns/triggerable.rb

```diff
@@ -39,8 +39,14 @@ def set_multibuild_flavor
     @multibuild_container = @package_name.gsub(/.*:/, '') if @package_name.present? && @package_name.include?(':')
   end
 
+  private
+
   def package_from_project_link?
     # a remote package is always included via project link
     !(@package.is_a?(Package) && @package.project == @project)
   end
+
+  def project_links_to_remote
+    @project.links_to_remote? || @project.scmsync.present?
```

Comment:
Could we just swap the checks? `scmsync` is just a db field, if it is present it will skip the call of `links_to_remote` which is more expensive it expands all the projects etc.

```suggestion
    @project.scmsync.present? || @project.links_to_remote?
```